suites/tipset/rewards_penalties.go

```diff
@@ -67,7 +69,7 @@ func TestMinerRewardsAndPenalties(t *testing.T, factory state.Factories) {
 				// No reward is paid to the miner directly. The funds for block reward were already held by the
 				// reward actor. The gas reward should be added to the treasury. The sum of block and gas reward
 				// should be chalked up to the miner address.
-				thisReward := big.Add(reward.BlockRewardTarget, gasSum)
+				thisReward := big.Add(BlockRewardTarget, gasSum)
```

Comment:
@anorth was `reward.BlockRewardTarget` the value of the block reward or something different? What do we call it now if not this?

Reply:
It was. You can now get the value of the prior block reward from the reward actor state.

The comment above here is obviously all wrong now: the reward is paid to the miner.